jsonnet/kube-prometheus/kube-prometheus-eks.libsonnet

```diff
@@ -1,63 +1,70 @@
-local k = import 'github.com/ksonnet/ksonnet-lib/ksonnet.beta.4/k.libsonnet';
-local service = k.core.v1.service;
-local servicePort = k.core.v1.service.mixin.spec.portsType;
-
 {
   _config+:: {
     eks: {
       minimumAvailableIPs: 10,
-      minimumAvailableIPsTime: '10m'
-    }
+      minimumAvailableIPsTime: '10m',
+    },
   },
   prometheus+: {
     serviceMonitorCoreDNS+: {
-        spec+: {
-          endpoints: [
-            {
-              bearerTokenFile: "/var/run/secrets/kubernetes.io/serviceaccount/token",
-              interval: "15s",
-              targetPort: 9153
-            }
-          ]
+      spec+: {
+        endpoints: [
+          {
+            bearerTokenFile: '/var/run/secrets/kubernetes.io/serviceaccount/token',
+            interval: '15s',
+            targetPort: 9153,
+          },
+        ],
+      },
+    },
+    AwsEksCniMetricService: {
```

Comment:
nit: we should change this to `serviceAwsEksCniMetric` as this is a more common pattern in the repository. For example we already have `serviceMonitorAwsEksCNI` and `serviceMonitorCoreDNS`

Reply:
I think this is better as a follow up, I wouldn't overlap the refactoring with "improvement" changes. If anything breaks then overlapping would cause a mess.